### ISSUE TITLE
Don't require aria-valuemin and aria-valuemax in aria-required-attr

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1500]
While these are required values, after some testing it turns out that NVDA and VoiceOver don't announce these values at all, and that while JAWS does claim it can, but it's hard to find out even how to get it to do that (Thanks @mfairchild365 for the research).

This needs to be removed from the required attributes for the following roles:
- scrollbar
- slider
- spinbutton

**Comment by chandana7393** [comment]
Working as expected no throwing any violation related to aria-valuemin and aria-valuemax.
![image](https://user-images.githubusercontent.com/45285854/58472811-b848c400-8164-11e9-8dc1-a1486109df89.png)
**Tested Environment:**
Axe-coconut - 3.8.0.14704v
Chrome - 74v
OS - Windows 10 64 bit.
